src/Host/Client/Impl/rtvs/R/packages.R

```diff
@@ -1,11 +1,40 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See LICENSE in the project root for license information.
 
-packages.installed <- function() {
+packages.installed.functions <- function() {
+	unname(lapply(packages.installed.named(), function(p) {
```

Comment:
AFAIK `unname` doesn't produce any errors, and `lapply` can produce error only if underlying function fails.